.gitignore

```diff
@@ -16,6 +16,7 @@
 # BookWyrm
 .env
 /images/
+/static/
```

Comment:
Hmmm the version I read was outdated and I actually don't love this...I should check upstream for what the goal was here.

Reply:
Confirmed this should be fine; it was added as part of https://github.com/bookwyrm-social/bookwyrm/commit/4d23edddca5692d7ef66f8b66b88e6f3bc061008